modules/nodes-application-secrets-using.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-application-secrets-creating-using-sa_{context}"]
+= Creating and using secrets
+
+As an administrator, you can create a service account token secret. This allows you to distribute a service account token to applications that must authenticate to the API.
+
+.Procedure
+
+. Create a service account in your namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create sa <service_account_name> -n <your_namespace>
+----
+
+. Create a `Secret` object to generate a service account token by running the following command:
++
+[source,yaml]
+----
+$ oc apply -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret_name> <1>
+  annotations:
+    kubernetes.io/service-account.name: "sa-name" <2>
+type: kubernetes.io/service-account-token <3>
+EOF
+----
+<1> Specifies a secret name.
+<2> Specifies an existing service account name. If you are creating both the `ServiceAccount` and the `Secret` objects, create the `ServiceAccount` object first.
+<3> Specifies a service account token secret.
+
+. Get the secret and store it in a `TOKEN` variable by running the following command:
++
+[source,terminal]
+-----
+$ TOKEN=$(oc get secret <sa_token_secret> -o jsonpath='{.data.token}' | base64 --decode) <1>
+-----
+<1> The `<sa_token_secret>` parameter specifies the service token secret.
+
+. Get the cluster API by running the following command:
++
+[source,terminal]
+----
+$ curl -X GET <openshift_cluster_api> --header "Authorization: Bearer $TOKEN" <1>
+----
+<1> The `<openshift_cluster_api>` parameter specifies the OpenShift cluster API.
```

Comment:
As detailed in my one of my preceding comments, you could add another call out to this line, as follows:

```
<2> Replace `<token>` with the service account token that is output in the preceding command.
```

Reply:
Addressed